docs/pywbemcli/generaloptions.rst

```diff
@@ -881,87 +881,57 @@ Output formats for groups and commands
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |centralinsts   | 'simple' | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  info         | 'simple' | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  interop      | 'text'   | text           | Alternate is table format                  |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  namespaces   | 'simple' | table          | Alternate is text format                   |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  profiles     | 'simple' | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|connection|  delete       | None     | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  export       | None     | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  list         | 'simple' | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  save         |  None    | table          |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  select       |  None    | None           |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  show         |  None    |                | Currently ignores output format            |
-+----------+---------------+----------+----------------+--------------------------------------------+
-|          |  test         |  None    | None           |                                            |
-+----------+---------------+----------+----------------+--------------------------------------------+
+default format for the :ref:`--output-format general option`.
+
+The following shows the default format for each command and the alternate
+formats where the values mean:
+
+objects: ``xml``, ``repr``, or ``txt``
+
+table: ``table``, ``plain``, ``simple``, ``grid``, ``psql``, ``rst``, or ``html``
+
+========== ============= ======== ============== ============================================
```

Comment:
This reformat resolves several issues.   Note that there is a table directive that puts a title on the table. I assume that it also allows creating references to the table but have not quite sorted that out.  Using that
directive for first time in pywbem doc update today. Turns out the old format really does not work well with the table directive so just changed the one in pywbem